src/Entity/RulesComponentAddForm.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules\Entity\RulesComponentAddForm.
+ */
+
+namespace Drupal\rules\Entity;
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides a form for adding rules components.
+ */
+class RulesComponentAddForm extends RulesComponentFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function actions(array $form, FormStateInterface $form_state) {
+    $actions = parent::actions($form, $form_state);
+    $actions['submit']['#value'] = $this->t('Add new rules component');
+    return $actions;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function save(array $form, FormStateInterface $form_state) {
+    parent::save($form, $form_state);
+
+    drupal_set_message($this->t('Rules component %label was created', array('%label' => $this->entity->label())));
```

Comment:
Not all actions in rules d7 have a message. For now we follow core (block and node) with this pattern: 
%type %label has been <action>.

Component %label has been deleted.